doc/source/user_guide/indexing.rst

```diff
@@ -867,7 +867,20 @@ Boolean indexing
 .. _indexing.boolean:
 
 Another common operation is the use of boolean vectors to filter the data.
-The operators are: ``|`` for ``or``, ``&`` for ``and``, and ``~`` for ``not``.
+The operators are: ``|`` for ``or``,  ``^`` for ``bitwise or``, ``&`` for ``and``, and ``~`` for ``not``. 
```

Comment:
This sentence seem redundant now with the table you provided (which is a lot better IMO)

Reply:
I reverted back to before my whatsnew commit. Then, I added and committed an example for  `&` and  removed the redundant sentence.

Sorry about the whatsnew! I saw that in the PR template and thought - for whatever reason - it was required.